auth-web/tests/unit/components/TransactionsDataTable.spec.ts

```diff
@@ -140,7 +143,8 @@ describe('TransactionsDataTable tests', () => {
     expect(row1Cells.at(4).text()).toBe('123')
     expect(row1Cells.at(5).text()).toBe('ab12')
     expect(row1Cells.at(6).text()).toBe('tester 123')
-    expect(row1Cells.at(7).text()).toContain('January 24, 20233:09 PM')
+    // Input:createdOn: new Date('2023-01-24T14:00:00')
+    expect(row1Cells.at(7).text()).toContain('January 24, 20232:00 PM')
```

Comment:
Oh wait a minute... Shouldn't this read January 24, 20236:00 AM ? Since the input is 2PM UTC?

Reply:
No, because newDate converts it into PST